src/DocsGenerator/DefaultPatterns.cs

```diff
@@ -0,0 +1,172 @@
+using System.Collections.Generic;
+
+namespace CodacyCSharp.DocsGenerator {
+    static class DefaultPatterns {
+        public static List<string> defaultPatterns = new List<string> {
```

Comment:
Codacy found an issue: [Change the visibility of 'defaultPatterns' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/codacy/codacy-sonar-csharp/pullRequest?prid=6187140)